### PR TITLE
feat: handle send bitcoin error

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM

**Comment by CharisSiarampalis** [comment]
@CedrikNikita I got this error when i chose slow transaction
image

**Comment by CedrikNikita** [comment]
> @CedrikNikita Lgtm I got this error when i chose slow transaction

This is the exact expected behavior

File: src/protocols/bitcoin/components/TransferReview.vue [hunk 1]
```diff
@@ -72,13 +72,22 @@ export default defineComponent({
...
       openDefaultModal({
         title: t('modals.transaction-failed.msg'),
         icon: 'critical',
+        msg,
       });
     }
 
+    // TODO: replace that
+    // This is happening because of inner TemplateRenderer issue on parsing the blocksteam response
+    function HtmlEncode(s: string) {
```
Comment (konradodo) [inline src/protocols/bitcoin/components/TransferReview.vue:85]: Can we named better that parameter?

**Review by konradodo — APPROVED** [review]

**Review by ifaouibadi — APPROVED** [review]

**Comment by subhod-i** [comment]
Nice work.

**Pull request merged**